.trivyignore

```diff
@@ -1 +1,2 @@
 CVE-2022-40897
+CVE-2023-39325
```

Comment:
Can you double check the .trivyignore file and remove whatever's not needed anymore?

Reply:
Already double checked, both are necessary

Reply:
How did you check - is there a run without them that fails? The action might have a bug then..

Reply:
Yes, I checked with the following runs: https://github.com/canonical/gunicorn-k8s-operator/actions/runs/7557994564/job/20578506931  https://github.com/canonical/gunicorn-k8s-operator/actions/runs/7557890775/job/20578171237